### ISSUE TITLE
Unique encryption of email

### ISSUE THREAD

**Issue by measwel** [brix/crypto-js#232]
Good day, 

I am currently encrypting my users email with:
  var cipher = crypto.createCipher(aes, config.private_key);
  var encrypted = cipher.update(clear_text, 'utf8', 'hex') + cipher.final('hex');
  return encrypted;

Is the encryption result going to be unique, so I can search a user's email based on the encrypted string? If not, what cipher or method should I use to guarantee that the result is always going to be unique?

**Comment by suddjian** [comment]
Yes, this is unique as long as all the emails are encrypted with the same private key.
